packages/x-date-pickers/src/locales/itIT.ts

```diff
@@ -0,0 +1,62 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { CalendarPickerView } from '../internals/models';
+
+const views = {
+  hours: 'ore',
+  minutes: 'minuti',
+  seconds: 'secondi',
```

Comment:
Do you want to apply the same trick from https://github.com/mui/mui-x/pull/5468/files#diff-bea2fa1f0c901abc3e771a07a2da9f52f0195ddd3e41bf7b2cf61a2f6353855fR6 here?

```suggestion
  hours: 'le ore',
  minutes: 'i minuti',
  seconds: 'i secondi',
```